helm/kubernetes-namespace-prep/templates/conjur-connect-configmap.yml

```diff
@@ -0,0 +1,21 @@
+---
+{{- if .Values.conjurConfigMap.create -}}
+{{- $g := (lookup "v1" "ConfigMap" .Values.authnk8s.namespace .Values.authnk8s.goldenConfigmap).data -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: conjur-connect-configmap 
+  labels: 
+    app.kubernetes.io/name: {{ .Values.authnk8s.namespace }}-authn-k8s
+    app.kubernetes.io/component: conjur-conn-configmap
+    app.kubernetes.io/instance: {{ .Values.authnk8s.namespace }}-authn-k8s
+    app.kubernetes.io/part-of: app-namespace-config
+    app.kubernetes.io/managed-by: helm
```

Comment:
Same as above, the `app.kubernetes.io/managed-by: helm` label is probably redundant since Helm adds its own similar label dynamically.